plugin/storage/integration/es_index_cleaner_test.go

```diff
@@ -33,7 +33,7 @@ const (
 	spanIndexName         = "jaeger-span-2019-01-01"
 	serviceIndexName      = "jaeger-service-2019-01-01"
 	indexCleanerImage     = "jaegertracing/jaeger-es-index-cleaner:latest"
-	rolloverImage         = "jaegertracing/jaeger-es-rollover:latest"
+	rolloverImage         = "jaegertracing/jaeger-es-rollover:1.57.0"
```

Comment:
Wow, I didn't know we do that. That's pretty bad, the tests should always be using _current_ code, i.e. we should just  run these directly (they are Go binaries, right?) or do a quick `docker build .` (it's lightweight, just copies the binary over).

Doesn't need to be in this PR, we can open help wanted ticket.

Reply:
yes, this is far from great. I can take a look at it in a separate PR

Reply:
https://github.com/jaegertracing/jaeger/issues/5709